<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
</head>
<body>

<h2>Description</h2>
<p>These tracks are miRNA expression levels of 78 primary cell types and 42 cancer/immortalized cell lines. Data was obtained from three sources.  The first source was from <a href="http://genome.cshlp.org/content/early/2017/09/06/gr.222067.117.long" target="_blank">McCall MN et al</a>.
 Cells were generally grown in culture or were flow sorted. The second source was <a href="https://www.nature.com/nbt/journal/v35/n9/full/nbt.3947.html" target="_blank">FANTOM5</a> data, which was obtained from commercially purchased primary cell lines grown in culture.
 The third source was from <a href="https://academic.oup.com/nar/article/4080663" target="_blank">Juzenas et al</a>. The isolation technique was magnetic activated cell sorting (MACS) from total blood.  From the first source, all samples had
 greater than 1 million miRNA reads.  For the second and third sources, all samples had > 500,000 miRNA reads. Each cell type may be the average of several similar cells or the data from
 a single cell line/culture. That can be noted on the more detailed graph. Some cell types sequenced by the FANTOM5 group will not appear here as they had <500,000 miRNA reads. In total, 545 RNA-seq data sets were used from across the sources. Data is expressed in reads per million miRNA reads (RPM) through analysis performed through the miRNA alignment software <a href="http://atlas.pathology.jhu.edu/baras/miRge.html">miRge</a>. This data is not otherwise normalized.  

<p>A separate processing step was carried out, although not part of the display, which normalized across the McCall et al. samples via the RUV (Remove Unwanted Variation) process. Please see Process 2 in
 the Methods section for more information. This data is not log normalized.</p>
<h2>Display Conventions and Configuration</h2>
	
<p>Samples are color coded, using the GTEx color palate to indicate similar cell types.  Individual cells can be toggled on/off using the "Go to Primary cells updated track controls" tool. The track is best viewed with the Log10(x+1)transform unselected and the view limits
maximum set to 50,000 RPM or similar.
</p>

<h2>Methods</h2>

<h3>Process 1</h3>

Data was obtained from the Sequence Read Archive from primary cells grown in culture or from flow sorted cells.  Fastq files were processed with <a href="http://atlas.pathology.jhu.edu/baras/miRge.html">miRge</a> for counting miRNAs. miRNAs were normalized to reads per million miRNA reads (RPM).

<h3>Process 2</h3>
	
Additional processing was performed with the Remove Unwanted Variation (RUV) method to normalize across samples. The miRNA matrix file with RUV values for each miRNA for each data sets can be found here: <a href="https://raw.githubusercontent.com/mhalushka/UCSC_miRNA_barchart/master/hg38/miRNA_primary_RUV_matrix.txt">Primary cells</a> and <a href="https://raw.githubusercontent.com/mhalushka/UCSC_miRNA_barchart/master/hg38/miRNA_cancer_RUV_matrix.txt">Cancer cells</a>. 

<h3>Notes</h3>
	
For a full description of the method, please see the methods section of the manuscripts below (McCall et al 2017, de Rie et al 2017, Juzenas et al, 2017).

<p></p>
	
</ul>

<h2>Data Access</h2>

All primary data is available through the Sequence Read Archive.  Specific sample information can be obtained through the manuscripts listed below. 	

<h2>Credits</h2>

<p>Arun H. Patil, Matthew N. McCall, Min-Sik Kim, Mohammed Adil, Yin Lu, Christopher J. Mitchell, Pamela Leal-Rojas, Jinchong Xu, Manoj Kumar, Valina L. Dawson, Ted M. Dawson, 
Alexander S. Baras, Avi Z. Rosenberg, Dan E. Arking, Kathleen H. Burns, Akhilesh Pandey, Marc K. Halushka.</p>
<p>
Arun and Marc thank the FANTOM5 team and the Hemmrich-Stanisak laboratory for making their data available in the public domain and Christopher Lee from the UCSC Genome Browser team for 
excellent technical assistance.

</p><p>

For inquiries, please contact Marc Halushka at halushka@jhmi.edu
	
</p>
	
<h2>References</h2>

<p>

1. Matthew N. McCall, Min-Sik Kim, Mohammed Adil, Arun H. Patil, Yin Lu, Christopher J. Mitchell, Pamela Leal-Rojas, Jinchong Xu, Manoj Kumar, Valina L. Dawson, Ted M. Dawson, Alexander S. Baras, Avi Z. Rosenberg, Dan E. Arking, Kathleen H. Burns, Akhilesh Pandey, Marc K. Halushka. Towards the human cellular microRNAome. <a href="http://genome.cshlp.org/content/early/2017/09/06/gr.222067.117.long" target="_blank">Genome Research</a>, 2017.

</p><p>
2. Derek de Rie, et al. An integrated expression atlas of miRNAs and their promoters in human and mouse <a href="https://www.nature.com/nbt/journal/v35/n9/full/nbt.3947.html" target="_blank">Nature Biotechnology</a>, 2017.</p>
<p>
3. Simonas Juzenas et al. A comprehensive, cell specific microRNA catalogue of human peripheral blood <a href="https://academic.oup.com/nar/article/4080663" target="_blank">Nucleic Acids Research</a>, 2017.
</p><p>
4. Baras AS, Mitchell CJ, Myers JR, Gupta S, Weng LC, Ashton JM, Cornish TC, Pandey A, Halushka MK. miRge - A Multiplexed Method of Processing Small RNA-Seq Data to Determine MicroRNA Entropy. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0143066" target="_blank">PLoS One</a>. 2015 Nov 16;10(11):e0143066.

</p>
	
</body>
</html>
